### DISCUSSION TITLE
使用[自定义顶栏][极简首页]时，拦截原有页面加载

### DISCUSSION THREAD

**Discussion by psychosispy** [the1812/Bilibili-Evolved#4564] [功能建议 / Ideas]
目前的版本使用以上两个功能进入首页时，原版顶栏会先闪一下，同时原版首页内容也会继续在后台加载，有没有可能拦截原版的这些加载，提升性能和美观，不知道技术上是否可以实现

**Comment by the1812** [comment]
不能, 脚本还没运行的时候原版就已经开始加载了
